orca-redis/src/main/java/com/netflix/spinnaker/orca/config/RedisConfiguration.java

```diff
@@ -35,6 +35,7 @@
 import rx.Scheduler;
 
 @Configuration
+@ConditionalOnProperty(value = "redis.enabled", matchIfMissing = true)
```

Comment:
This is the biggest change in this PR. We didn't have a mechanism for avoiding this configuration entirely, for whatever reason.